terraform/route53.tf

```diff
@@ -0,0 +1,171 @@
+locals {
+  host_bastion = "34.254.208.229"
+  host_chef    = "46.4.67.10"
+  host_www     = "54.217.220.47"
+  host_status  = "138.201.32.77"
+}
+
+resource "aws_route53_zone" "nixos" {
+  name = "nixos.org"
+}
+
+resource "aws_route53_record" "nixos-mx" {
+  zone_id = "${aws_route53_zone.nixos.zone_id}"
+  name    = ""
+  type    = "MX"
+  ttl     = "14300"
+
+  records = [
+    "10 mx00.udag.de.",
+    "20 mx01.udag.de.",
+  ]
+}
+
+resource "aws_route53_record" "nixos-a" {
+  zone_id = "${aws_route53_zone.nixos.zone_id}"
+  name    = ""
+  type    = "A"
+  ttl     = "500"
+  records = ["${local.host_www}"]
+}
+
+resource "aws_route53_record" "nixos-bastion" {
+  zone_id = "${aws_route53_zone.nixos.zone_id}"
+  name    = "bastion"
+  type    = "A"
+  ttl     = "600"
+  records = ["${local.host_bastion}"]
+}
+
+resource "aws_route53_record" "nixos-discourse" {
+  zone_id = "${aws_route53_zone.nixos.zone_id}"
+  name    = "discourse"
+  type    = "CNAME"
+  ttl     = "3600"
+  records = ["nixos1.hosted-by-discourse.com"]
+}
+
+resource "aws_route53_record" "nixos-cache" {
+  zone_id = "${aws_route53_zone.nixos.zone_id}"
+  name    = "tarballs"
+  type    = "CNAME"
+  ttl     = "3600"
+  records = ["dualstack.v2.shared.global.fastly.net"]
+}
+
+resource "aws_route53_record" "nixos-cache-verification" {
+  zone_id = "${aws_route53_zone.nixos.zone_id}"
+  name    = "_d47a77b375708cea087182ee599174c0.cache"
+  type    = "CNAME"
+  ttl     = "600"
+  records = ["_f437d0ffb520b017f4d72beb71afedf8.acm-validations.aws"]
```

Comment:
Do we actually need to keep these after validation has succeeded?

Reply:
I think they re-validate every time they re-issue the certificate.

Reply:
I am not sure. It's better to keep them around for now.